src/more-infos/more-info-updater.html

```diff
@@ -0,0 +1,55 @@
+<link rel="import" href="../../bower_components/polymer/polymer.html">
+<link rel="import" href="../../bower_components/paper-button/paper-button.html">
+
+<dom-module id="more-info-updater">
+  <style>
+    .data-entry .value {
+      max-width: 200px;
+    }
+    .center {
+      text-align: center;
+    }
+    .centered {
+      margin: auto;
+    }
+    .divider {
+      padding: 5px;
+    }
+    .link {
+      color: #03A9F4;
+    }
+  </style>
+  <template>
+    <div class='layout vertical'>
+
+      <!-- GitHub Link -->
+      <div class='data-entry layout justified horizontal center' style="width:100%" id='setting'>
+        <paper-button class='centered link' on-tap="linkTapped">
+          View on GitHub
+        </paper-button>
+      </div>
+
+      <div class='data-entry layout justified horizontal'>
+        <div class='key'>Updated On:</div>
+        <div class='value'>[[stateObj.attributes.date]]</div>
+      </div>
+
+      <div class='data-entry layout justified vertical'>
```

Comment:
The release notes header now will still show up on the more info card even if stateObj.attributes.message does not exist.

![avkhgia](https://cloud.githubusercontent.com/assets/1444314/8681080/35ae9600-2a1a-11e5-8930-452ae1063e8a.png)